overview.md

```diff
@@ -6,18 +6,19 @@ The Endgame Plan is a proposal to overhaul and improve the governance and tokeno
 
 The Endgame Plan was [introduced](https://forum.makerdao.com/t/decentralized-voter-committee-on-tuesday-5pm-cest-all-mkr-holders-welcome/15166) by Rune Christensen, the co-founder of Maker. After a series of initial discussions, Maker Governance accepted the [MIP set](https://vote.makerdao.com/polling/QmTmS5Nf) ratifying this plan on 24 October 2022. 
 
-The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). Note that some parts are now out of date.  
+The Endgame Plan remains in active development, led by Rune Christensen. The latest complete and published description of the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). The Endgame Constitution, covering the  proposed by Christensen is available [here](https://forum.makerdao.com/t/mip101-the-maker-constitution/19621). Note that some parts are now out of date.  
```

Comment:
Seems like you're missing a word here? 'covering the proposed by Christensen'

Reply:
Meant to remove that sentence, done now.